src/pages/Admin/BackgroundSelect/BackgroundSelect.scss

```diff
@@ -11,25 +11,35 @@
   background-color: $dark;
   background-position: center;
 
+  &__maps-header {
+    font-size: $font-size--md;
+    font-weight: normal;
+    color: rgba(255, 255, 255, 0.5);
+    margin: $spacing--xxl auto $spacing--md auto;
+  }
+
   &__map-grid {
-    display: grid;
-    width: 50%;
-    grid-gap: $spacing--md;
-    grid-template-columns: repeat(3, 1fr);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    width: 100%;
   }
 
   &__map {
-    width: 100%;
-    padding-bottom: 50px;
+    width: 23%;
```

Comment:
You are explicitly setting the percentage and people will get confused, you should bring back the old logic and change the repeat to `(4, 1fr)` and just in increase `width: 50%` to something bigger. Grid makes it much more obvious that only 4 elements will be rendered on a row.

Reply:
But it will put a fixed space between them which make them look bad
I will put back 25 %

Reply:
fixed in 21dc9b1e0834b6219b5ccf8a32fca56a6a2484c6